pytket/extensions/qiskit/backends/ibmq_emulator.py

```diff
@@ -108,9 +108,14 @@ def backend_info(self) -> BackendInfo:
     def required_predicates(self) -> List[Predicate]:
         return self._ibmq.required_predicates
 
-    def default_compilation_pass(self, optimisation_level: int = 2) -> BasePass:
+    def default_compilation_pass(
+        self, optimisation_level: int = 2, placement_options: Optional[Dict] = None
+    ) -> BasePass:
+        """
+        See for :py:meth:`IBMQBackend.default_compilation_pass`.
```

Comment:
```suggestion
        See documentation for :py:meth:`IBMQBackend.default_compilation_pass`.
```

Reply:
Oops, this didn't save properly.

Done